src/api.js

```diff
@@ -17,11 +47,27 @@ const requestPagePreview = ( lang, title, isTouch, callback, request = cachedReq
 				type: data.type
 			}
 		}
+		// special case: there is no summary but there is a description
+		if ( data.type === 'no-extract' && data.description ) {
+			return {
+				title: data.titles.canonical,
+				extractHtml: '<p>' + data.description + '</p>',
+				imgUrl: data.thumbnail ? data.thumbnail.source : null,
+				dir: data.dir,
+				type: 'standard'
+			}
+		}
 		return false
 	}, callback )
 
 }
 
+const requestPagePreview = ( lang, title, isTouch, callback, request = cachedRequest ) => {
+	return title.indexOf( ':' ) === -1 ?
+		requestPcsSummary( lang, title, isTouch, callback, request ) :
+		requestMwExtract( lang, title, isTouch, callback, request )
```

Comment:
Looks like `isTouch` is not really needed for either `requestMwExtract` or `requestPcsSummary`?

Reply:
Right. I removed it.